### DISCUSSION TITLE
Add custom language

### DISCUSSION THREAD

**Discussion by kiyoungsong** [dream-num/Luckysheet#1243] [Q&A]
Hello everyone,
I want to add Korean language for luckysheet
How could i apply it?
Any ideas?

**Comment by luncheon** [comment]
[feat: allow library users to add locales #1055](https://github.com/dream-num/Luckysheet/pull/1055) was merged, but not published to npm. If you build the latest Luckysheet yourself, you can add locales.

**Reply by kiyoungsong** [reply]
Hello luncheon.
Thanks for about that.
I have a question if i want to add custom languages then do i have to build Luckysheet myself?
Also i am using that in React project so do you know how to import ./plugins/js/plugin.js and luckysheet.umd.js file in react??

**Reply by luncheon** [reply]
- https://www.npmjs.com/package/luckysheet has not been updated for two years. Perhaps an unofficial build may exist. ([I personally have an unofficial custom build](https://github.com/luncheon/luckysheet-nightly), but it is for my own use and I have no plans to document or support it.) I don't know which is harder, using unofficial builds or building your own.
- React has nothing to do directly with importing. Import resolution depends on the module bundler (webpack, rollup, vite, esbuild, etc.) (or the browser).
